web/pages/Chat/components/InputBar.tsx

```diff
@@ -161,74 +165,72 @@ const InputBar: Component<{
         onSubmit={() => send()}
         cleared={cleared}
       />
-      <div>
```

Comment:
Removed this wrapper div as it does not seem to serve any purpose that I could tell and was otherwise another component I had to set an explicit height on.